python/ray/exceptions.py

```diff
@@ -394,6 +403,21 @@ def __init__(self, cause: Union[RayTaskError, ActorDiedErrorContext] = None):
     def from_task_error(task_error: RayTaskError):
         return ActorDiedError(task_error)
 
+    @staticmethod
+    def _error_msg_from_node_death_info(node_death_info: NodeDeathInfo) -> str:
+        if node_death_info.reason == NodeDeathInfo.EXPECTED_TERMINATION:
+            prefix = "\tthe actor's node was terminated intentionally: "
+        elif node_death_info.reason == NodeDeathInfo.UNEXPECTED_TERMINATION:
+            prefix = "\tthe actor's node was terminated unexpectedly: "
+        elif node_death_info.reason == NodeDeathInfo.AUTOSCALER_DRAIN_PREEMPTED:
+            prefix = "\tthe actor's node was preempted: "
+        elif node_death_info.reason == NodeDeathInfo.AUTOSCALER_DRAIN_IDLE:
+            prefix = "\tthe actor's node was drained due to idle: "
```

Comment:
yep. I was thinking what message should be given to the user in case a bad system state leads to this, and make it distinguishable from other cases. An error message in another error message does not seem reasonable.
Removed now.